### DISCUSSION TITLE
Once you enable the flipper globally, you can't disable it for a particular actor. Is this intentional? If yes, why.

### DISCUSSION THREAD

**Discussion by bonniesimon** [jnunemaker/flipper#875] [Q&A] [answered]
When I enable a flipper for all actors, i.e flipper_gates(key=boolean & value=true), then I can't disable the flipper for a particular team. Is this intentional? If yes, then what is the reasoning behind this? I'm just trying to understand this design.

**Comment by jnunemaker** [comment, marked as answer]
Yes, this is intentional. You can read more here: https://github.com/flippercloud/flipper/issues/514. There is some commentary and other issues that have been referenced. 

It's not a simple change as it ripples all the way into the adapters. I've been considering looking into adding it as part of the pro gem since it would add complication but has been requested. 

Closing this to keep things tidy but happy to chat more! 

If you don't mind, I would love to know what you are using flipper for as well and how it is working out for you (what went well, what was hard other than what you mentioned here).

If you aren't comfortable leaving that information here in a public comment, feel free to email me directly john@fewerandfaster.com. I'm just always curious. :)
